packages/next-polyfill-nomodule/src/index.js

```diff
@@ -12,6 +12,23 @@ import 'core-js/features/array/of'
 import 'core-js/features/function/has-instance'
 import 'core-js/features/function/name'
 import 'core-js/features/map'
+import 'core-js/features/math/acosh'
```

Comment:
Math polyfills were excluded purposefully as they increase size by quite a bit and are very unlikely to be used in Next.js applications.

Reply:
Sounds reasonable. I thought that a ~1KB change in bundle size wouldn't matter, but I could revert these changes if desired.

Reply:
I think a warning should be emitted during development whenever `Math.{acosh,asinh,…}` gets called and no polyfill has been loaded for them. Until a better solution arises, a ~1KB increase in legacy bundle sizes seem to be a tradeoff worth considering. Other prototype-modifying <=ES2019 library features are supported out of the box.

Reply:
We've had very few reports from people that needed it, so it seems that this should be based on usage instead of always including it.

Reply:
Alright, I'm removing this from the PR, then 😊
All the other issues have been fixed for your requests.